docs/fields.yaml

```diff
@@ -0,0 +1,129 @@
+# Fields definition, used to generate documentation
```

Comment:
Can we add this to the static-frontend-config ?

I would like to get rid of `ipfix.ts` file from console plugin repo in the end. 
We can also rely on these definitions to check if field is a label and add types since the query engine check for numbers.

Reply:
I can surely move that to the config, ok, but it will stay unused in code for the time being, right? at least for this PR

Reply:
well then I can also change my `flow-format-validation` script into a go test